source/retryable-writes/tests/README.md

```diff
@@ -106,17 +106,197 @@ Drivers should test that transactions IDs are always included in commands for su
 
 The following tests ensure that retryable writes work properly with replica sets and sharded clusters.
 
-1. Test that retryable writes raise an exception when using the MMAPv1 storage engine. For this test, execute a write
-   operation, such as `insertOne`, which should generate an exception. Assert that the error message is the replacement
-   error message:
+- Test that retryable writes raise an exception when using the MMAPv1 storage engine. For this test, execute a write
+  operation, such as `insertOne`, which should generate an exception. Assert that the error message is the replacement
+  error message:
 
-   ```
-   This MongoDB deployment does not support retryable writes. Please add
-   retryWrites=false to your connection string.
-   ```
+  ```
+  This MongoDB deployment does not support retryable writes. Please add
+  retryWrites=false to your connection string.
+  ```
 
-   and the error code is 20.
+  and the error code is 20.
 
-   [!NOTE]
-   storage engine in use MAY skip this test for sharded clusters, since `mongos` does not report this information in its
-   `serverStatus` response.
+  > [!NOTE]
+  > Drivers that rely on `serverStatus` to determine the storage engine in use MAY skip this test for sharded clusters,
+  > since `mongos` does not report this information in its `serverStatus` response.
```

Comment:
These admonitions don't render within other elements, so I think this would be better written as a basic paragraph with no "Note" lead-in. Rendering issues aside, I think that makes more sense since the paragraph is providing direction for the test implementation by use of "MAY".

See [this discussion](https://github.com/orgs/community/discussions/16925#discussioncomment-9156825) and/or [this test file](https://github.com/sinsukehlab/NOTE-test/blob/main/NOTE.md) for more info on rendering issues.

Reply:
They're left-aligned now.